packages/xo-server/src/xo-mixins/http.mjs

```diff
@@ -12,16 +16,14 @@ export default class Http {
     return this._agent
   }
 
-  constructor(_, { config: { httpProxy = firstDefined(process.env.http_proxy, process.env.HTTP_PROXY) } }) {
-    this._hasOwnHttpProxy = httpProxy != null
+  constructor(_, { config: { httpProxy } }) {
+    if (httpProxy !== undefined) {
+      this._hasOwnHttpProxy = true
```

Comment:
what use is this var ?

Reply:
Used by the internal plugin `xo-server-xoa`.